snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -386,7 +387,10 @@ def ignore(directory, files):
         shutil.copytree(self.code.sourcedir, self.code.build_basedir,
                         symlinks=True, ignore=ignore)
 
+        script_runner = ScriptRunner(builddir=self.code.build_basedir)
```

Comment:
@kyrofa yes it is, it feels safer to have people type `$whatever-subdir/something` than `../someother-thing`